client/src/modules/Course/Components/ReviewModal.tsx

```diff
@@ -115,9 +138,31 @@ const ReviewModal = ({
         major: selectedMajors,
       }
       submitReview(newReview)
+    }
+  }
 
-      console.log('Submitting')
-    } else return
+  // Handle click of submit button
+  function onSubmitReview() {
+    if (!noReviews && isLoggedIn) {
+      handleSubmitReview()
+      signIn('profile')
+    } else {
+      handleSubmitReview()
+      setAnonymousOpen(true)
+      setOpen(false)
```

Comment:
I'd rename this to `setReviewOpen` to avoid confusion as to what component is opened.